os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -319,11 +319,6 @@ void amebasmart_mount_partitions(void)
 #endif /* CONFIG_FLASH_PARTITION */
 }
 
-void amebasmart_memory_initialize(void)
-{
-
-}
-
```

Comment:
Please tell us why memory_initialize and all its references have been removed for amebasmart?

Reply:
Hi @amogh-samsung ,

That is referenced from imx6 during early bringup stage, but we do not need that, thus it should be removed.
I will filter out the cleanup changes to another PR.

Reply:
Yes please, thankyou